intelij-plugin/src/main/java/tech/pantheon/yanginator/plugin/completion/YangPsiTreeChangeListener.java

```diff
@@ -24,40 +24,33 @@
 
     @Override
     public void childrenChanged(@NotNull PsiTreeChangeEvent event) {
-        PsiElement prevPsiElement = getPrevPsiElement(event);
-        if (event.getFile() != null) {
-            if (PsiEditorUtil.findEditor(event.getFile().getNode().getPsi()) != null) {
-                POP_UP.setPrefixMatcher(prevPsiElement == null ? "" : prevPsiElement.getText());
-            }
+        if (event.getFile() == null ||
+                event.getFile().getNode() == null ||
+                event.getFile().getNode().getPsi() == null){
+            return;
+        }
+        PsiElement prevPsiElement = getPrevPsiElement(event.getFile().getNode().getPsi());
+        if (PsiEditorUtil.findEditor(event.getFile().getNode().getPsi()) != null) {
+            POP_UP.setPrefixMatcher(prevPsiElement == null ? "" : prevPsiElement.getText());
         }
     }
 
     @Nullable
-    private PsiElement getPrevPsiElement(@NotNull PsiTreeChangeEvent event) {
-        PsiElement currentPsiElement = getCurrentPsiElement(event);
-        if (event.getFile() != null) {
-            return event.getFile().getNode().getPsi()
-                    .findElementAt(getOffsetOfCaret(event) - (currentPsiElement == null ? 1 : currentPsiElement.getTextLength()));
-        }
-        return null;
+    private PsiElement getPrevPsiElement(@NotNull PsiElement element) {
+        PsiElement currentPsiElement = getCurrentPsiElement(element);
+        return element.findElementAt
```

Comment:
One liner is probably better than this, or split it in different way but not on (